frontend/public/components/monitoring/query-browser.tsx

```diff
@@ -154,8 +154,10 @@ const SpanControls: React.FC<SpanControlsProps> = React.memo(
 );
 
 const TOOLTIP_MAX_ENTRIES = 20;
-const TOOLTIP_MAX_WIDTH = 300;
+const TOOLTIP_MAX_WIDTH = 400;
```

Comment:
Will this run offscreen for phone-sized displays? (Maybe we had this problem anyway?)

Reply:
It won't go off the screen because this is just setting a max width. If the graph is small, the tooltip adjusts to the fit the available width.

However, your comment did prompt me to reduce the 2 `TOOLTIP_MAX_*_JUT_OUT` constants slightly, which looks a little better at phone sizes.